src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -530,5 +583,6 @@ export default connect(
     storageDomains: state.storageDomains,
     maxNumOfVmCpus: state.config.get('maxNumOfVmCpus', 384),
     maxMemorySizeInMiB: 4194304, // TODO: 4TiB, no config option pulled as of 2019-Mar-22
+    config: state.config,
```

Comment:
Follow the same pattern as for `maxNumOfVmCpus` and pull the value out of `config` instead of pushing the entire config object down.

The more we can minimize redux/immutablejs code in components themselves, the easier it is to refactor the store in future.